src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,11 +2,13 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
     private ProductParser productParser;
     private FileReaderService fileReaderService;
```

Comment:
Where's the @Inject annotation

Reply:
To this and all comments after this: on QnA we discussed that issue. I have used different approach and since code works mentors just need to take a closer look and decide.